src/backend/InvenTree/machine/tests.py

```diff
@@ -275,11 +275,6 @@ def test_print_label(self):
         self.assertEqual(self.print_labels.call_args.args[0], self.machine.machine)
         self.assertEqual(self.print_labels.call_args.args[1], template)
 
-        # TODO re-activate test
-        # self.assertQuerySetEqual(
-        #    self.print_labels.call_args.args[2], parts, transform=lambda x: x
-        # )
-
```

Comment:
I think this is still relevant, when this is implemented again. Currently somehow this was removed from the function implementation, but I still see high value to have direct access to the class instances of the specific items that where to print.